src/description.cpp

```diff
@@ -441,6 +442,18 @@ void Description::Entry::parseSdpLine(string_view line) {
 	}
 }
 
+void Description::Media::addSSRC(uint32_t ssrc, std::string name) {
+    mAttributes.emplace_back("ssrc:" + std::to_string(ssrc) + " cname:" + name);
```

Comment:
Just a suggestion here: instead of comparing the attribute strings, you can store the ssrcs as integers and parse them like it's done for the direction.

Reply:
Absolutely! I have plans to rewrite this very soon, just wanted to confirm it works.